spinalcordtoolbox/deepseg/inference.py

```diff
@@ -178,7 +187,7 @@ def segment_monai(path_img, tmpdir, predictor):
     return [fname_out], [target]
 
 
-def segment_nnunet(path_img, tmpdir, predictor):
+def segment_nnunet(path_img, tmpdir, predictor, device):
```

Comment:
Ditto for the expected type of `device` here, especially since it's unused, so it's hard to infer.

Actually, maybe add a note that `device` is unused, and only present so that `segment_nnunet` has the same function signature as `segment_monai`?

I guess Monai has no mechanism to specify the device?
```suggestion
def segment_nnunet(path_img, tmpdir, predictor, device: torch.device):
```